packages/integration-karma/test/template/attribute-capture/x/test/test.html

```diff
@@ -0,0 +1,4 @@
+<template>
+    <!-- capture="true" or hidden="false" is a compilation error (to be fixed) -->
```

Comment:
Let's remove this comment since it's a little confusing without the proper context, and we already have an issue for it.